conans/client/packager.py

```diff
@@ -42,13 +49,17 @@ def create_package(conanfile, pkg_id, source_folder, build_folder, package_folde
     output.info("Package folder %s" % package_folder)
 
     try:
-        package_output = ScopedOutput("%s package()" % output.scope, output)
-        output.highlight("Calling package()")
         conanfile.package_folder = package_folder
         conanfile.source_folder = source_folder
         conanfile.install_folder = install_folder
         conanfile.build_folder = build_folder
 
+        plugin_manager.execute("pre_package", conanfile=conanfile, conanfile_path=conanfile_path,
+                               reference=str(reference), package_id=pkg_id)
+
+        package_output = ScopedOutput("%s package()" % output.scope, output)
```

Comment:
This `package_output` is not being used!

Reply:
Yes, it is only used for the copy report some lines below

Reply:
Yes, it is only used for the copy report some lines below

Reply:
uupss! 😓